docs/csharp/language-reference/keywords/method-parameters.md

```diff
@@ -10,20 +10,101 @@ ms.assetid: 680e39ff-775b-48b0-9f47-4186a5bfc4a1
 ---
 # Method Parameters (C# Reference)
 
-Parameters declared for a method without [in](./in-parameter-modifier.md), [ref](./ref.md) or [out](./out-parameter-modifier.md), are passed to the called method by value. That value can be changed in the method, but the changed value will not be retained when control passes back to the calling procedure. By using a method parameter keyword, you can change this behavior.  
-  
- This section describes the keywords you can use when declaring method parameters:  
-  
+In C#, arguments can be passed to parameters either by value or by reference. Remember that C# types can be either reference types (`class`) or value types (`struct`):
+
+- *Pass by value* means **passing a copy of the variable** to the method.
+- *Pass by reference* means **passing access to the variable** to the method.
+- A variable of a *reference type* contains a reference to its data.
+- A variable of a *structure type* contains its data directly.
```

Comment:
```suggestion
- A variable of a *value type* contains its data directly.
```

I think this should be "value type", I've never heard of "structure type" before this PR. I always thought it was just value or reference types.